index.html

```diff
@@ -28,12 +28,200 @@ <h1>I'm David <br>Glad to see you!</h1>
+                </li>
+                <li class="techstack">
+                    <div>
+                        <ul>
+                            <li class="canopy">CANOPY</li>
+                            <li class="bed">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="aboutProject">
+                    A daily selection of privately <br> personalized reads: no accounts or <br> sign-ups required.
+                </li>
+                <li class="lang">
+
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="js">javaScript</li>
+                    </ul>
+
+                </li>
+                <li class="view">
+                    <a href="">
+                        <h3>See Project</h3>
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="aboutme">
+        <div class="aboutInfo">
+            <h2>About <br> Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+                <h3>
+                    LET'S CONNECT
+                </h3>
+                <ul class="icons">
+                    <li><a href="./index.html"><img src="./images/Vector.svg" alt="twitter icon"></a></li>
+                    <li><a href="./index.html"><img src="./images/Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                    <li><a href="./index.html"><img src="./images/Vector (1).svg" alt=""></a></li>
+                    <li><a href="./index.html"><img src="./images/Vector (2).svg" alt="github icon"></a></li>
+                    <li><a href="./index.html"><img src="./images/Vector (3).svg" alt=""></a></li>
+                </ul>
+                <h3 class="get">
+                    Get my resume
+                </h3>
+        </div>
+        <div class="langInfo">
+            <h3>Languages</h3>
+            <ul>
+                <li class="pLangs javaScript">
+                    <img class="image" src="./images/Ellipse 1 (1).png"></img>
+                    <h2>JavaScript</h2>
+                </li>
+                <li class="pLangs markup">
+                    <img class="image" src="./images/Ellipse 1 (2).png"></img>
+                <h2>HTML</h2>
+                </li>
+                <li class="pLangs cascade">
+                    <img class="image" src="./images/Ellipse 1 (2).png"></img>
+                <h2>CSS</h2>
+                </li>
+            </ul>
+        </div>
+        <div class="frameInfo">
+            <h2>Frameworks</h2>
+            <h2>Skills</h2>
+        </div>
+    </section>
```

Comment:
- [ ] It's awesome that you make technology in each category(e.g., "JavasScript", "HTML" and "CSS") a `ul` with each being inside `li`, But we also need to follow the same structure for technology categories ("Languages", "Frameworks" and "Skills"). Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
- [ ] Also please add the arrows to your technologies as shown in the Figma design. This is to make sure you have a design identical to the Figma design.
![Screenshot 2023-01-13 211034](https://user-images.githubusercontent.com/69077061/212390390-b6283273-b83c-47a2-aad1-029e1c3a811f.png)